src/django/api/management/commands/direct_data_load.py

```
@@ -0,0 +1,443 @@
+                break
+
+
+def mark_row(
+        service,
+        spreadsheet_id,
+        tab_id,
+        row_index,
+        num_columns,
+        col_index,
+        message,
+        format={}):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)